kratos/tests/cpp_tests/processes/test_calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -1176,10 +1617,9 @@ namespace Testing {
                 }
             }
         }
-        // For embedded geometry on fluid element edges no cuts are detected
-        // --> (TODO) solution: detect as "standard" wall in ComputeEdgeIntersection in CalculateDiscontinuousDistanceToSkinProcess?
-        KRATOS_CHECK_EQUAL(n_incised, 0);
-        KRATOS_CHECK_EQUAL(n_intersected, 0);
+
+        KRATOS_CHECK_EQUAL(n_incised, 2);
+        KRATOS_CHECK_EQUAL(n_intersected, 1);
```

Comment:
This test correponds to the following situation:

![image](https://user-images.githubusercontent.com/32136457/116856025-aa2a3a80-abfa-11eb-91fb-14fff4326de1.png)

With the new changes, the zero-intersctions are considered and an epsilon is applied, so the new intersections "looks" like this:

![image](https://user-images.githubusercontent.com/32136457/116856101-ccbc5380-abfa-11eb-8608-45925e0d7034.png)

Now, elem 1 and 5 are incised (n_incised = 2) and elem 2 is intersected (n_intersected = 1)